### PR TITLE
Tune webpack config and add Dockerfile

### PR DESCRIPTION
**Pull request by jakemcdermott**
Add Dockerfile for running containerized dev server. Update webpack config to make dev server available over exposed docker port.

### CODE REVIEW

File: awx/ui_next/README.md [hunk 1]
```diff
@@ -27,3 +27,14 @@ To run a single test (in this case the login page test):
 * `npm test -- src/screens/Login/Login.test.jsx`
 
 **note:** Once the test watcher is up and running you can hit `a` to run all the tests
+
+
+## Container
+
+To run the dev server in a container:
```
Comment (jlmitch5) [inline awx/ui_next/README.md]: Could you provide some short context about why you'd want to do this (or not)?

File: awx/ui_next/_Dockerfile.Temp [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# This is not part of the PR - debugging something
```
Comment (jlmitch5) [inline awx/ui_next/_Dockerfile.Temp]: I guess we'd want to remove this or add to .gitignore?

**Review by jlmitch5 — CHANGES_REQUESTED** [review]
This is cool, though I'm not entirely sure when running in a container would be a benefit...would be cool to call out why/how.

Also do we want to add or update any root makefile targets to utilize this?

File: awx/ui_next/_Dockerfile.Temp (hunk 2 above)
Comment (jakemcdermott) [reply awx/ui_next/_Dockerfile.Temp]: oh no I intend to remove this file before merging, @dsesami are currently debugging something and using this as a tool.

File: awx/ui_next/README.md (hunk 1 above)
Comment (jakemcdermott) [reply awx/ui_next/README.md]: @jlmitch5 It's beneficial when running in CI systems and automation servers. There's no huge benefit to using this for local development at this time and I wouldn't recommend it.

I'll update the readme with something mentioning CI.

Comment (jakemcdermott) [reply awx/ui_next/README.md]: alright @jlmitch5 I've added some notes mentioning , CI servers, zuul, etc. and also a sentence about this not being intended for general development.

**Review by jlmitch5 — APPROVED** [review]
talked about things offline, looks good

**Comment by jakemcdermott** [comment]
recheck

**Review by dsesami — APPROVED** [review]

**Pull request merged**